recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "2.4.rc3" %}
+{% set source_tag = "2.4-rc3" %}
+
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/{{ source_tag }}.tar.gz
+  sha256: 48a52af8f9da6deef5c01d26a30ba4bdc19929999d6680dab6461b92b5a63bd7
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  features:
+    - vc14  # [win]
+
+requirements:
+  build:
+    - cmake >=3.8
+    - {{ compiler('cxx') }}  # [not osx]
+    - clang_osx-64  # [osx]
+    - clangxx_osx-64  # [osx]
+
+  host:
+    - python
+    - numpy
+    - swig
+    - hdf5 1.8.18|1.8.18.*
+    - zlib
+    - libmatio
+    - openblas
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - hdf5 1.8.18|1.8.18.*
```

Comment:
Sorry if I missed this conversation, why do we need this older version of `hdf5`? Is something requiring it?

Reply:
yes, if I understand the error properly, libmatio requires it. See https://github.com/conda-forge/staged-recipes/pull/5534#discussion_r197161939)

Reply:
Ah ok. Thanks. Sorry I missed that.